src/main/java/io/pravega/perf/PerfStats.java

```diff
@@ -202,12 +244,40 @@ private long windowTimeMS(long time) {
         }
     }
 
+    /***
+     * Perf Recorder that records the number of bytes every second.  Percentiles are reported in human form, i.e. MB/s
+     */
+    @NotThreadSafe
+    static private class CSVThroughputWriter  {
+        final private CSVPrinter csvPrinter;
+
+        CSVThroughputWriter(String action, String csvFile) throws IOException {
+            csvPrinter = new CSVPrinter(Files.newBufferedWriter(Paths.get(csvFile)), CSVFormat.DEFAULT
+                .withHeader("Start", "End","Events",action+" Events Throughput", "Bytes", action+" MiB Throughput"));
```

Comment:
Minor: while we do not added checkstyle in this repo yet, one of the existing rules we apply in other repos is to leave a space after each argument and before/after symbols (like `+`).